syntax = "proto3";

package DBAccount;




message DBUserAccount
{
	string user_id = 1;
	string account_name = 2; //用户名
	string password = 3;
	string password_hash = 4;
	string session_key = 5;
	string v = 6;
	string s = 7;
	string register_time = 8;
	string last_ip = 9;
	string last_login_time = 10;
	string os = 11;
	bool is_banned = 12;
}


message RoleInfo
{
	int32 server_id = 1;	// 服务器ID
	string server_name = 2; // 服务器名字
	string role_name = 3; // 角色名字
}

message DBRoleInfoList
{
	repeated RoleInfo role_list = 1;	// 角色列表
}


